java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -41,12 +40,49 @@ public class CommandTests {
 
     private static final String INITIAL_VALUE = "VALUE";
 
-    public static final List<String>
-        DEFAULT_INFO_SECTIONS = List.of("Server", "Clients", "Memory", "Persistence", "Stats", "Replication", "CPU", "Modules", "Errorstats", "Cluster", "Keyspace");
-    public static final List<String> EVERYTHING_INFO_SECTIONS = REDIS_VERSION.feature() >= 7
-        // Latencystats was added in redis 7
-        ? List.of("Server", "Clients", "Memory", "Persistence", "Stats", "Replication", "CPU", "Modules", "Commandstats", "Errorstats", "Latencystats", "Cluster", "Keyspace")
-        : List.of("Server", "Clients", "Memory", "Persistence", "Stats", "Replication", "CPU", "Modules", "Commandstats", "Errorstats", "Cluster", "Keyspace");
+    public static final List<String> DEFAULT_INFO_SECTIONS =
+            List.of(
+                    "Server",
+                    "Clients",
+                    "Memory",
+                    "Persistence",
+                    "Stats",
+                    "Replication",
+                    "CPU",
+                    "Modules",
+                    "Errorstats",
+                    "Cluster",
+                    "Keyspace");
```

Comment:
Did this change...?  I don't understand why you have so many spotless differences in this file.

Reply:
Oh... this is the file that failed spotless before.  This will get fixed by rebase, right?

Reply:
Rebased and cleaned